irohad/ordering/impl/ordering_service_impl.hpp

```diff
@@ -39,44 +46,40 @@ namespace iroha {
      * @param delay_milliseconds timer delay
      * @param max_size proposal size
      */
-    class OrderingServiceImpl
-        : public proto::OrderingService::Service,
-          network::AsyncGrpcClient<google::protobuf::Empty> {
+    class OrderingServiceImpl : public network::OrderingService {
      public:
       OrderingServiceImpl(
           std::shared_ptr<ametsuchi::PeerQuery> wsv, size_t max_size,
-          size_t delay_milliseconds);
-
-      grpc::Status SendTransaction(
-          ::grpc::ServerContext *context, const protocol::Transaction *request,
-          ::google::protobuf::Empty *response) override;
-
-      ~OrderingServiceImpl() override;
+          size_t delay_milliseconds,
+          std::shared_ptr<network::OrderingServiceTransport> transport);
 
-     private:
       /**
        * Process transaction received from network
        * Enqueues transaction and publishes corresponding event
        * @param transaction
        */
-      void handleTransaction(model::Transaction &&transaction);
+      void onTransaction(const model::Transaction& tx) override;
```

Comment:
Make consistent name and doc-name.

Reply:
the prefix is 'on' because this method is invoked as soon as transaction arrives

Reply:
In this case, rework doc with `tx` name.